### ISSUE TITLE
Prevent screen readers from reading the arrow emoji in menu items

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-mu-plugins#453]
See https://github.com/WordPress/wporg-parent-2021/pull/94, https://github.com/WordPress/wporg-main-2022/issues/311

It was requested that the arrow emojis be wrapped in `aria-hidden`, so that they're not announced to screen reader users. The code added there does not apply to the header & footer nav menus.

Since more of these will be introduced in #450, we should try to do the same for the nav items.

Example of unwrapped arrow:
<img width="547" alt="Screenshot 2023-09-20 at 11 31 07 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/541093/ba93b980-450f-4e39-8d69-75f4508236d0">
